### PR TITLE
Use Requests.ts and .env on frontend

### PR DESCRIPTION
**Pull request by DrewThomasCorps**

### CODE REVIEW

File: client/src/components/MapContainer.tsx [hunk 1]
```diff
@@ -22,14 +24,11 @@ const MapContainer = () => {
...
         getPorts();
         updateVesselPositions();
-        // eslint-disable-next-line react-hooks/exhaustive-deps
     },[]);
 
-    const getPorts = () => {
-        fetch(`http://localhost:3001/ports?mapview_1=1`)
-            .then(response => response.json())
-            // .then(data => setPorts(data));
-            .then(data => setPorts(mapPorts(data)));
```
Comment (Rekkisftw) [inline client/src/components/MapContainer.tsx:32]: I had trouble finding the right dependency for a useEffect that would do the port location math based on the background tile after getting ports from the backend. That's why I was using setPorts(mapPorts(data)) here.

I believe with your implementation, the ports will be set, but their positions will not be calculated for the background image boundaries.

**Review by Rekkisftw — CHANGES_REQUESTED** [review]

File: client/src/components/MapContainer.tsx (hunk 1 above)
Comment (Rekkisftw) [reply client/src/components/MapContainer.tsx:32]: I know getting the data, and changing the data shouldn't be coupled. This isn't your problem, it's mine. I'd be happy to get suggestions.

TLDR: Ports that come from the DB must be enriched with xPosition and yPosition based on background tile. This change doesn't take that into account.
